src/main/java/com/marginallyclever/makelangelo/plotter/plottercontrols/MarlinInterface.java

```diff
@@ -103,14 +105,21 @@ private void onClose() {
 	}
 	
 	private void onTimeoutCheck() {
-		if (System.currentTimeMillis() - lastReceivedTime > TIMEOUT_DELAY) {
-			logger.trace("Heartbeat: M400");
-			chatInterface.sendCommand("M400");
+		long delay = System.currentTimeMillis() - lastReceivedTime;
+		if (delay > TIMEOUT_DELAY) {
+			if (delay > TIMEOUT_DELAY * 5) {
```

Comment:
this should probably be a constant like `TIMEOUT_DELAY`

Reply:
I have updated the code